package.json

```diff
@@ -242,12 +242,12 @@
         "docker.defaultRegistry": {
           "type": "string",
           "default": "",
-          "description": "Default registry when tagging an image, empty string will target Docker Hub when pushing."
+          "description": "(Deprecated) Default registry when tagging an image, empty string will target Docker Hub when pushing."
         },
         "docker.defaultRegistryPath": {
           "type": "string",
           "default": "",
-          "description": "Path within registry to push to."
+          "description": "Default registry and path when tagging an image, empty string will target Docker Hub when pushing"
```

Comment:
empty string part no longer applies

Reply:
The docker command still assumes no prefix as pushing to Dockerhub. 
![image](https://user-images.githubusercontent.com/30420973/44601658-0a058d80-a792-11e8-9645-f2cb3ebf94a5.png)

Does it make sense to expose Docker's assumption as part of this statement?

Reply:
Okay, I guess that makes sense.